roles/lmod/tasks/main.yml

```diff
@@ -0,0 +1,77 @@
+#
+# ansible role to install lmod:
+#
+---
+- name: "install packages"
```

Comment:
@ajdecon , These are both one liners, do you think they need to be split up into rhel/ubuntu yml files?

Reply:
Not necessarily. Probably fine to just have them as single tasks with a conditional. The file split just makes it cleaner to add to later.

Reply:
I am missing the RH counter part of those.